### DISCUSSION TITLE
我想问下如果我本地部署了archery，并且想测试集成oidc，在oidc服务商配置的archery回调地址应该是什么呢

### DISCUSSION THREAD

**Discussion by arthas3014** [hhyo/Archery#2587] [Q&A]
我想问下如果我本地部署了archery，并且想测试集成oidc，在oidc服务商配置的archery回调地址应该是什么呢

**Comment by LeoQuote** [comment]
路由是 /oidc/callback 

参考 https://mozilla-django-oidc.readthedocs.io/en/stable/installation.html

**Reply by arthas3014** [reply]
我在本地使用docker-compose构建并启动了，然后在oidc服务商配置回调地址为:  http://127.0.0.1:9123/oidc/callback/
也在.env文件中配置了相关oidc的环境变量，但是archery启动后马上报错重启
`ENABLE_OIDC=True
OIDC_RP_WELLKNOWN_URL=https://anycross.feishu.cn/sso/xxxxxxx/openid-configuration
OIDC_RP_CLIENT_ID=xxxxxx
OIDC_RP_CLIENT_SECRET=xxxxxxxx`
日志报错为: 
`ile "/opt/archery/archery/settings.py", line 297, in <module>
    OIDC_OP_LOGOUT_ENDPOINT = config["end_session_endpoint"]
KeyError: 'end_session_endpoint'` 
请问是哪里配置错了吗

**Reply by arthas3014** [reply]
你好可以添加一个联系方式吗，交流比较方便些可以吗

**Reply by LeoQuote** [reply]
确实有bug https://github.com/hhyo/Archery/blob/c83850d83538fbd712a2a07776f557c5b069dab6/archery/settings.py#L318C39-L318C59 应该是飞书的 oidc 实现不标准加上我们这边的代码不健壮导致的, 我建议你提一下pr, 或者找飞书让他们完善一下 well-known 接口

**Reply by arthas3014** [reply]
没有这个config["end_session_endpoint"]，能正常使用这个oidce登陆吗，要是不影响的话我直接注释掉？

**Reply by LeoQuote** [reply]
比较建议将这系列的 `["end_session_endpoint"]` 都改为 `.get("end_session_endpoint")` , 你可以改改看, 有空可以提个 pr

**Reply by LeoQuote** [reply]
我也不太清楚能不能正常登录, 你可以试试看
